tests/flakes/check.sh

```diff
@@ -25,6 +25,18 @@ EOF
 
 (! nix flake check $flakeDir)
 
+cat > $flakeDir/flake.nix <<EOF
+{
+  outputs = { self, ... }: {
+    overlays.x86_64-linux.foo = final: prev: {
+    };
+  };
+}
+EOF
+
+checkRes=$(nix flake check $flakeDir 2>&1 && fail "nix flake check --all-systems should have failed" || true)
+echo "$checkRes" | grepQuiet "error: overlay is not a function, but a  instead"
```

Comment:
```suggestion
echo "$checkRes" | grepQuiet "error: overlay is not a function, but a set instead"
```

Reply:
Let's see if this fixes the test.

Reply:
Oh no.... I changed the string to be explicitly sure the test actually fails and then I squashed this into the commits as well :facepalm: 

Well this is embarrassing... sorry.

Reply:
Don't worry about it. It's part of being efficient.
We all make such mistakes and that's fine because we have CI. All part of the process.